nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/dependencies/DependencyResolution.java

```diff
@@ -252,6 +252,17 @@ public boolean requireChanges() {
         return res;
     }
 
+    /**
+     * @since 1.4.26
+     */
+    public List<String> getInstallPackageNames() {
```

Comment:
add a comment: it seems to be a merge and sort of `getLocalPackagesToInstall()` + `getLocalPackagesToInstall()`